core/sql/regress/seabase/EXPECTED011

```diff
@@ -299,8 +299,10 @@ LC   RC   OP   OPERATOR              OPT       DESCRIPTION           CARD
 LC   RC   OP   OPERATOR              OPT       DESCRIPTION           CARD
 ---- ---- ---- --------------------  --------  --------------------  ---------
 
-1    .    2    root                  o         x                     2.00E+000
-.    .    1    trafodion_delete                T011T1                2.00E+000
+3    .    4    root                            x                     2.00E+000
```

Comment:
I was surprised to see plan change for deleting very few number of rows.  Sounds like we need to run some performance tests?